sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/controllers/GroundingController.java

```diff
@@ -102,7 +102,7 @@ Object searchInDocuments(
     final var messages =
         results.getResults().stream()
             .flatMap(resultsInner1 -> resultsInner1.getResults().stream())
-            .flatMap(result -> result.getDataRepository().getDocuments().stream())
+            .flatMap(result -> result.getDocuments().stream())
```

Comment:
I just generate the code in JS and found out that this is a mistake from the spec. The server response still contain `dataRepository` level. I informed them in the [PR](https://github.tools.sap/AI/rage-proxy-inference/pull/388#issuecomment-13486386)